packages/react-native-sdk/src/components/Call/CallLayout/CallParticipantsSpotlight.tsx

```diff
@@ -113,6 +113,7 @@ export const CallParticipantsSpotlight = ({
                   callParticipantsSpotlight.spotlightContainer,
                 ]
           }
+          objectFit="contain"
```

Comment:
dont we need to set as contain only for screenshare tracks? here it is also set for video tracks